src/encoding/armor.js

```diff
@@ -246,7 +244,7 @@ function splitChecksum(text) {
 
   if (lastEquals >= 0) {
```

Comment:
Last two characters are `=\n`, so we want `lastEquals !== text.length - 2`, which will end up being the `=` we're looking for.

To be safe, I'd do `lastEquals >= 0 && lastEquals < text.length - 2`, in case future changes elsewhere cause the `\n` to miss at the end. Tried it & tests are passing.

What do you think?